### PR TITLE
multiple GA tracker support

### PR DESCRIPTION
**Pull request by jpslav**
Adds support for multiple GA trackers.  Sends tracking commands to each tracker found under `window.ga`.  

The BE PR that adds multiple trackers is https://github.com/openstax/tutor-server/pull/1410.  This PR can be used with or without that PR.

### CODE REVIEW

File: tutor/specs/helpers/analytics.spec.js [hunk 1]
```diff
@@ -23,12 +23,52 @@ describe('Analytics', function() {
...
+    // Set up a mock GA function/object
+    function Tracker(name) {
+      this.name = name;
+      this.get = function(fieldName) {
+        if ('name' == fieldName) { return this.name; }
+      };
+    }
+
+    var mockGa = function() {
+      if (arguments.length == 0) { return; }
```
Comment (nathanstitt) [inline tutor/specs/helpers/analytics.spec.js]: not a big deal at all, but `var` usage is frowned on with modern js.   Typically you use `const` for variables that won't be re-assiged and `let` for when they will be.

Awesome work on figuring out the stubs and putting them to good use 😁

**Comment by philschatz** [comment]
Just sayin there's https://github.com/philschatz/fetch-vcr . I can add an option for `XMLHttpRequest` if Google Analytics still uses that instead of `fetch`

**Comment by jpslav** [comment]
Cool @philschatz. Agreed probably not needed here (esp since we're not even really caring here about the response and just want to know that we are making the right calls per the docs).  But in general I love VCR approaches :-D

**Comment by jpslav** [comment]
@nathanstitt thanks re `var` `const` `let`.  I updated the PR.

File: tutor/specs/helpers/analytics.spec.js (hunk 1 above)
Comment (jpslav) [inline tutor/specs/helpers/analytics.spec.js]: @nathanstitt thanks re `var` `const` `let`.  I updated the PR.

Comment (jpslav) [inline tutor/specs/helpers/analytics.spec.js]: Also re stubs shout out to @philschatz who gave me late night help on that :-D

**Pull request merged**